app/src/frontend/building/sidebar.css

```diff
@@ -34,6 +34,14 @@
     text-decoration: none;
     color: #222;
     padding: 0.75rem 0.25rem 0.5rem 0;
+    z-index: 1000;
+}
+
+@media (min-width: 768px) {
+    .section-header {
+        position: sticky;
+        top: 0;
+    }
```

Comment:
On larger viewports where space is not an issue, making the header sticky makes navigating between categories easier